---
title: 例外処理ステートメント - C# リファレンス
ms.custom: seodec18
ms.date: 07/20/2015
helpviewer_keywords:
- keywords [C#], exception handling
ms.assetid: 6d0323a3-3164-411c-9b84-a3606bd0e13c
ms.openlocfilehash: ecb4b09e46df2159de5e68d94b588f32e621fa52
ms.sourcegitcommit: 2701302a99cafbe0d86d53d540eb0fa7e9b46b36
ms.translationtype: HT
ms.contentlocale: ja-JP
ms.lasthandoff: 04/28/2019
ms.locfileid: "64596669"
---
# <a name="exception-handling-statements-c-reference"></a><span data-ttu-id="82c24-102">例外処理ステートメント (C# リファレンス)</span><span class="sxs-lookup"><span data-stu-id="82c24-102">Exception Handling Statements (C# Reference)</span></span>
<span data-ttu-id="82c24-103">C# では、プログラムの実行時に発生する例外と呼ばれる異常事態を処理する組み込みサポートが用意されています。</span><span class="sxs-lookup"><span data-stu-id="82c24-103">C# provides built-in support for handling anomalous situations, known as exceptions, which may occur during the execution of your program.</span></span> <span data-ttu-id="82c24-104">このような例外は、通常の制御フローの外部にあるコードによって処理されます。</span><span class="sxs-lookup"><span data-stu-id="82c24-104">These exceptions are handled by code that is outside the normal flow of control.</span></span>  
  
 <span data-ttu-id="82c24-105">ここでは、例外処理に関する次のトピックについて説明します。</span><span class="sxs-lookup"><span data-stu-id="82c24-105">The following exception handling topics are explained in this section:</span></span>  
  
- [<span data-ttu-id="82c24-106">throw</span><span class="sxs-lookup"><span data-stu-id="82c24-106">throw</span></span>](../../../csharp/language-reference/keywords/throw.md)  
  
- [<span data-ttu-id="82c24-107">try-catch</span><span class="sxs-lookup"><span data-stu-id="82c24-107">try-catch</span></span>](../../../csharp/language-reference/keywords/try-catch.md)  
  
- [<span data-ttu-id="82c24-108">try-finally</span><span class="sxs-lookup"><span data-stu-id="82c24-108">try-finally</span></span>](../../../csharp/language-reference/keywords/try-finally.md)  
  
- [<span data-ttu-id="82c24-109">try-catch-finally</span><span class="sxs-lookup"><span data-stu-id="82c24-109">try-catch-finally</span></span>](../../../csharp/language-reference/keywords/try-catch-finally.md)  
  
## <a name="see-also"></a><span data-ttu-id="82c24-110">関連項目</span><span class="sxs-lookup"><span data-stu-id="82c24-110">See also</span></span>

- [<span data-ttu-id="82c24-111">C# リファレンス</span><span class="sxs-lookup"><span data-stu-id="82c24-111">C# Reference</span></span>](../../../csharp/language-reference/index.md)
- [<span data-ttu-id="82c24-112">C# プログラミング ガイド</span><span class="sxs-lookup"><span data-stu-id="82c24-112">C# Programming Guide</span></span>](../../../csharp/programming-guide/index.md)
- [<span data-ttu-id="82c24-113">C# のキーワード</span><span class="sxs-lookup"><span data-stu-id="82c24-113">C# Keywords</span></span>](../../../csharp/language-reference/keywords/index.md)
- [<span data-ttu-id="82c24-114">ステートメントのキーワード</span><span class="sxs-lookup"><span data-stu-id="82c24-114">Statement Keywords</span></span>](../../../csharp/language-reference/keywords/statement-keywords.md)
- [<span data-ttu-id="82c24-115">例外と例外処理</span><span class="sxs-lookup"><span data-stu-id="82c24-115">Exceptions and Exception Handling</span></span>](../../../csharp/programming-guide/exceptions/index.md)
